app/assets/stylesheets/application.scss

```diff
@@ -74,3 +74,13 @@ ul {
   margin-right: 5px;
   width: 50px;
 }
+
+#search {
+  width:200px;
+  background-color:#333333;
```

Comment:
Color literals like `#333333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space